client-tutorial/README.md

```diff
@@ -0,0 +1,114 @@
+
+Note the patient resources make heavy use of FHIR extensions such as geolocation, for the purposes of the demo. for example as below:
+
+```json
+{
+  "name": [
+    {
+      "given": [
+        "Marilyn"
+      ],
+      "family": "Salazar"
+    }
+  ],
+  "gender": "female",
+  "address": [
+    {
+      "use": "home",
+      "city": "Boston",
+      "line": [
+        "518 James St. "
+      ],
+      "type": "postal",
+      "state": "MA",
+      "country": "USA",
+      "postalCode": "02212"
+    }
+  ],
+  "birthDate": "1918-10-09",
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/geolocation",
+      "extension": [
+        {
+          "url": "latitude",
+          "valueDecimal": 42.446396
+        },
+        {
+          "url": "longitude",
+          "valueDecimal": -71.459405
+        }
+      ]
+    }
+  ],
+  "identifier": [
+    {
+      "id": "48632",
+      "type": {
+        "id": "http://terminology.hl7.org/CodeSystem/v2-0203",
+        "text": "MR"
+      },
+      "value": "48632",
+      "system": "http://https://mimic.physionet.org/identifiers/subjectid"
+    }
+  ],
+  "resourceType": "Patient"
+}
+```
+
+
+## Database
+The database used in this demo is provided in SQL files for MySql (or MariaDB if you prefer). In MySQL it assumes version >8 syntax but would likely work fine on version 5 servers (untested). Could this database work in Postgres/DB2/SQLServer/oracle/etc? of course, the datbase files are pretty basic SQL. The inital creation script is called *lfh_demodb.sql* and then you simply apply the the scripts in order (if you don't know how to import scripts into mysql the easiest way (if you aren't running a graphical client such as DBeaver or PhpMyAdmin is to import from the command line client as follows:
+            `mysql -u<your_user_name> -p kafkaFhirDemoDb < 1.sql`
+note there are no spaces between `-u` and your username. The -p does support password entry in the command but that is a huge security risk as you password is now in your shell history (it will even chastise you). You can of course rename the database from *kafkaFhirDemoDb* but you will need to edit *lfh_demodb.sql* and the python code (specifically in *DatabaseUtil.createConnectionPool()*
+
+Note: of course you will need to edit the database url *DatabaseUtil.createConnectionPool()* to match your environment, including username/password and IP address. You of course need to specify in your mysql server that the user you select can access from an outside host if you are not running all this on localhost.
+
+## Running
+Assuming you got the docker stuff above launched, you now need to run EntryPoint.py to start the application. It will launch a desktop application called "LfhTutorial" with the strange Kivy GUI (see below)
+
```

Comment:
At this point, I'd expect to see a code block that shows the command needed to `run EntryPoint.py to start the application`.  Also, it helps to indicate where the user should be (top-level directory, etc) when they run the command.

Reply:
Wouldn't the convention to run a class be to be in the directory containing the file?